notebooks/05_transform.ipynb

```diff
@@ -0,0 +1,3444 @@
+{
```

Comment:
<p>Is it possible to do it using <code>dataset.transform()</code> like the others? For example, <code>val_dataset.transform("split", task="segmentation", splits=splits)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>It works as following your comment. Thanks.</p>